lawnchair/src/app/lawnchair/search/data/SearchRequest.kt

```diff
@@ -4,42 +4,56 @@ import android.content.Context
 import android.provider.ContactsContract
 import android.provider.MediaStore
 import android.util.Log
-import java.io.BufferedReader
-import java.io.InputStreamReader
-import java.net.HttpURLConnection
-import java.net.URL
+import app.lawnchair.search.data.suggestion.StartPageService
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
 import kotlin.coroutines.resume
 import kotlin.coroutines.suspendCoroutine
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.ResponseBody
 import org.json.JSONArray
 import org.json.JSONObject
+import retrofit2.Response
+import retrofit2.Retrofit
+
+val json = Json { ignoreUnknownKeys = true }
+
+val retrofit: Retrofit = Retrofit.Builder()
+    .baseUrl("https://www.startpage.com/")
+    .addConverterFactory(json.asConverterFactory("application/json".toMediaType()))
```

Comment:
And this https://github.com/LawnchairLauncher/lawnchair/blob/7aa0b7172c75000bfd3e113cc009f52dab9f979a/lawnchair/src/app/lawnchair/bugreport/KatbinService.kt#L20